src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -678,6 +677,44 @@ void convertExportAssignment(
     }
   }
 
+  /**
+   * Returns true is the object is an object literal where all values are symbols that match the
+   * name:
+   *
+   * <p>Ex: {A, B: B} -> true {A: C} -> false {A: A + 1} -> false
+   *
+   * <p>TODO(rado): see if we can also support simple renaming objects like {NewName: OldName}.
+   */
+  private boolean isObjLitWithSimpleRefs(Node node) {
+    if (!node.isObjectLit()) return false;
+    for (Node child : node.children()) {
+      if (!child.isStringKey() || !child.getFirstChild().isName()) {
+        return false;
+      }
+      if (!child.getString().equals(child.getFirstChild().getString())) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  private void moveExportStmtToADeclKeyword(
+      Node assign, Node exprNode, ExportedSymbol symbolToExport) {
+    // Rewrite the AST to export the symbol directly using information from the export
```

Comment:
Should this be a comment for the function, instead of an implementation comment?